drivers/usb/device/usb_dc_native_posix.c

```diff
@@ -384,7 +377,7 @@ int usb_dc_ep_read_wait(uint8_t ep, uint8_t *data, uint32_t max_data_len,
 	}
 
 	/* Check if OUT ep */
-	if (USBIP_EP_ADDR2DIR(ep) != USB_EP_DIR_OUT) {
+	if (USB_EP_GET_DIR(ep) != USB_EP_DIR_OUT) {
```

Comment:
s/a

Reply:
No, I will not change this logic here. Also not relevant to this PR.